CMakeLists.txt

```diff
@@ -60,15 +61,21 @@ pkg_check_modules(
   accountsservice
   libayatana-common
 )
-include_directories(${SOUNDSERVICE_INCLUDE_DIRS})
 
-pkg_check_modules(LOMIRI_API liblomiri-api>=0.1.1)
-pkg_check_modules(LOMIRI_SCHEMAS lomiri-schemas)
```

Comment:
Why have you dropped lomiri-schemas from REQUIRED pkg_check_modules? You have only re-added liblomiri-api below in the if(ENABLE_LOMIRI_FEATURES) below.

Reply:
> Why have you dropped lomiri-schemas from REQUIRED pkg_check_modules? You have only re-added liblomiri-api below in the if(ENABLE_LOMIRI_FEATURES) below.

The build system does not pick up `lomiri-schemas` and `lomiri-sounds`. We cannot use those two as dependencies.

Reply:
@tari01 Maybe we should adjust lomiri-schemas and lomiri-sounds then? It would definitely be really handy to use pkg-config for the detection of the presence of the two named packages.

Reply:
> @tari01 Maybe we should adjust lomiri-schemas and lomiri-sounds then? It would definitely be really handy to use pkg-config for the detection of the presence of the two named packages.

I will add the dependencies after these:
https://gitlab.com/ubports/core/lomiri-schemas/-/merge_requests/6
https://gitlab.com/ubports/core/lomiri-sounds/-/merge_requests/11

Reply:
The Lomiri pkgbuild files have been merged.